pep-0480.txt

```diff
@@ -236,56 +258,63 @@ End-to-End Signing
 ==================
 
 End-to-end signing allows both PyPI and developers to sign for the metadata
-downloaded by clients.  PyPI is trusted to make uploaded projects available to
-clients (PyPI signs the metadata for this part of the process), and developers
-sign the distributions that they upload to PyPI.
-
-In order to delegate trust to a project, developers are required to submit a
-public key to PyPI.  PyPI takes the project's public key and adds it to parent
-metadata that PyPI then signs.  After the initial trust is established,
-developers are required to sign distributions that they upload to PyPI using
-the public key's corresponding private key.  The signed TUF metadata that
-developers upload to PyPI includes information like the distribution's file
+downloaded by clients. PyPI is trusted to make uploaded projects available to
+clients (PyPI signs for the metadata for this part of the process), and
+developers sign for the distributions that they upload to PyPI.
+
+In order to delegate trust to a project, developers are required to submit an
+OpenID Connect identity or a public key to PyPI. PyPI takes the project's OIDC
+identity or public key and adds it to parent metadata, which PyPI then signs.
```

Comment:
Not sure I follow what's being suggested here.

Reply:
Ditto -- is the OIDC lifecycle specified here intended to be different from the one we're developing for credential-less publication? My understanding (which could be very wrong!) was that we were planning to use that one lifecycle for both.

For context, here's the authentication lifecycle as we're currently planning it: https://github.com/pypa/warehouse/issues/10619

Reply:
What is the project's OIDC "identity" vs the "public key"? This is totally unclear to me right now.